cloudhub-2/modules/ROOT/pages/ps-config-app-level-egress.adoc

```diff
@@ -0,0 +1,171 @@
++ 
+You can delete rules by clicking the delete button image:trash-icon.png[Delete button, 2%,2%].
+. Click *Update Egress Rules* to save your changes.
++
+Your changes will affect all the applications in the current private space.
++
+[NOTE]
+You can neither rename nor delete the default rule group.
+
+image::ps-egress-edit-default.png[Edit the default app level egress rule group]
+
+To create a new custom egress rule group:
+
+. Click *Create Rule Group*.
++
+The rule group configuration window opens.
+. Add a group name.
++ 
+You can edit and rename custom rule groups. 
+. In the *Egress Rules* section, click *Add Rule* and use the dropdown list to select the desired protocol and destination.
+. Click *Add Rule* to add new rules to the new rule group.
++ 
+You can delete rules by clicking the delete button image:trash-icon.png[Delete button, 2%,2%].
+. Click *Create Egress Rules* to save your changes.
++
+To use your custom rule groups, you must select them in the applications settings.
+
+To delete an existing custom rule group: 
+
+. Select *Delete* from the *More actions* menu image:more-icon.png[More actions menu, 2%,2%]. 
+. When prompted, click *Delete Rule Group* to confirm the deletion.
++
+[NOTE]
+Once you delete a custom rule group, any applications using it immediately starts using the default egress rule group instead.
+
+image::ps-egress-delete-ruleset.png[Delete custom egress rule group]
+
+You can disable the application level egress rule feature by clicking the feature's toggle. When prompted, click *Delete and Disable App-Level Rules* to confirm. 
+
+[NOTE]
+Once you disable the feature, all the custom app-level egress rule groups are deleted, and all app-level egress is allowed for all applications.
+
+== Apply Application Level Egress Rules When Deploying Your Applications
+
+You can apply the app-level egress rules that you configured in your private space when you deploy an application:
+
+. When deploying an application to your private space, go to the *Firewall Rules* tab.
++
+The *Egress Rule Group* section shows all the rule groups configured for the selected private space.
+. Use the dropdown list to select the egress ruleset you want to apply to your application. 
++
+The selected rule group's details appear below the *Egress Rule Group* list.
+. Click *Apply Changes* to apply the selected egress rules to your application.
+
+
+== Configure Application Egress Connection to Anypoint MQ or ObjectStore v2
+
+To set up connections to Anypoint MQ or OSv2, and access external endpoints, such as `anypoint.mulesoft.com` , follow the steps provided in this section.
+
+[NOTE]
+You must also allow outbound access to all 443 at the private space level separately.
+
+To allow HTTPS connections to Anypoint MQ, add an app-level egress rule group with the following two rules:
+
+. Allow egress to Anypoint control plane:
+** Protocol: `HTTPS`
+** Destination: `Custom Domains`
+*** With value
```

Comment:
If Protocol, Designation, and With value are interface items, they should be in bold.